dcos/dcos-ui
# DCOS-57287: User create modal does not clear password errors on close

Clear passwords errors for users when we
close the modal.

Closes https://jira.mesosphere.com/browse/DCOS-57287

## Testing
1. Go to Users tab.
2. Open the create user modal.
3. Try to create a new user, but write the "confirm password" wrong.
4. Close the modal and open it again.
5. Verify that no errors are shown.

## Trade-offs
None.

## Dependencies
https://github.com/mesosphere/dcos-ui-plugins-private/pull/1052

## Screenshots
### Before
![Peek 2019-08-02 14-20](https://user-images.githubusercontent.com/40791275/62457884-5ddf6b80-b784-11e9-8003-04bfac8ee9ce.gif)

### After
![Peek 2019-08-05 13-25](https://user-images.githubusercontent.com/40791275/62457992-8ff0cd80-b784-11e9-8c0d-271a18835fe0.gif)
